pkgs/by-name/sq/sqlelf/package.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, python3Packages
+, fetchPypi
+
+# ls and cat binaries are referenced in tests
+, coreutils
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "sqlelf";
+  version = "0.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-S2D918B/zuIpoF/FBTQKMjmPLjXt11wnxXMQ5rZrp7s=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    capstone
+    lief
+    apsw
+    sh
+  ];
+
+  nativeCheckInputs = with python3Packages; [
+    pytestCheckHook
+  ];
+
+  # Those tests depend on ls and cat being x86_64 Linux ELF files.
+  # The x86_64 check should be relaxed once sqlelf supports other architectures.
+  disabledTestPaths = lib.optionals (!stdenv.isLinux || !stdenv.isx86_64) [
+    "tests/test_sql.py"
+  ];
+
+  pytestFlagsArray = [
+    "-m" "not\\ slow"
+  ];
+
+  preCheck = ''
+    for test in tests/*; do
```

```suggestion
    for test in tests/*.py; do
```

Is probably a little more robust.